bin/eslint-dependency-graph.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const {CLIEngine} = require('eslint')
+
+// TODO: Figure out how to deactive other rules.
```

Comment:
Here's an issue, running `CLIEngine` rules with all rules enabled. This is going to slow down the evaluation of all files. We really only need to inject our dependency graph rule. However, we can't disable all configuration. We want to use whatever `parser` and other configuration that would affect how projects parse babel features, flow and jsx.